apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -12,18 +12,19 @@ defmodule Aecore.Miner.Worker do
   alias Aecore.Structures.Header
   alias Aecore.Structures.Block
   alias Aecore.Pow.Cuckoo
-  alias Aecore.Keys.Worker, as: Keys
   alias Aecore.Structures.TxData
   alias Aecore.Structures.SignedTx
   alias Aecore.Chain.ChainState
   alias Aecore.Txs.Pool.Worker, as: Pool
   alias Aeutil.Bits
   alias Aecore.Peers.Worker, as: Peers
+  alias Aecore.Wallet.Worker, as: Wallet
 
   require Logger
 
   @mersenne_prime 2147483647
   @coinbase_transaction_value 100
+  @wallet_default_pass "1234"
```

Comment:
If this is used in multiple locations then maybe it should be stored in the config or have it as an attribute in one module and have a getter function for it.

Reply:
Maybe store it in the config, as it is used both in the code and the tests.

Reply:
Yup, on it.